src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/dashboard/dashboard.component.scss

```diff
@@ -1,3 +1,7 @@
 main {
   padding-top: 20px;
 }
+
+.col-wider {
+  width: 9.5%;
+}
```

Comment:
why's this?

Reply:
and specifically to 9.5%?

Reply:
It was to align the button with the refresh-selector, no longer needed as the button has been replaced on the navbar